FuncGameDev/FSharp/Unity/Updater/CameraUpdater.fs

```diff
@@ -0,0 +1,23 @@
+﻿module CameraUpdater
+
+open UnityEngine
+
+let update =
+    // update camera
+    let camera = Camera.main
+    let transform = camera.transform
+
+    let cPosX = transform.position.x |> float
+    let cPosY = transform.position.y |> float
+    let pos = (cPosX, cPosY)
+
+    let cRotX = transform.rotation.x |> float
+    let cRotY = transform.rotation.y |> float
+    let rot = (cRotX, cRotY)
+
+    let cScaleX = transform.localScale.x |> float
+    let cScaleY = transform.localScale.y |> float
+    let scale = (cScaleX, cScaleY)
+
+    CameraBehavior.updateCamera pos rot scale 20.0 10.0 |> Commands.addCommand
```

Comment:
We should add this command where we add the other commands in the update loop. We shouldn't be adding commands in our gameobject updaters.